grammars/ruby.cson

```diff
@@ -1940,6 +1952,10 @@
     'name': 'punctuation.separator.key-value'
   }
   {
+    'match': '->'
+    'name': 'keyword.operator.lambda.ruby'
```

Comment:
OK - I've added a4f42cf455f105e6fbe4cc2d2bcaf428dccc28a4 to change `->` to match `lambda`. After looking at it further, it seems that all the other Kernel#methods are also in `support.function.kernel.ruby` so it makes sense to keep `lambda` in that category as well.